src/renderer/prefComponents/imageUploader/index.vue

```diff
@@ -53,16 +53,32 @@
             <el-button size="mini" :disabled="githubDisable" @click="setCurrentUploader('github')">Set as default</el-button>
           </div>
         </el-tab-pane>
+        <el-tab-pane label="Command line script" name="cliScript">
+          <div class="description">The script will be executed with the image file path as its only argument and it should output with any valid value for the <code>src</code> attribute of a <em>HTMLImageElement</em>.</div>
+          <div class="form-group">
+            <div class="label">
+              Script absolute path
```

Comment:
How about "Shell script location" because the script must be included in the PATH environment variable or absolute? Below too.